### PR TITLE
nf-test zip

### PR DESCRIPTION
**Pull request by JoseEspinosa**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/zip/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// nf-core modules test zip
```
Comment (sateeshperi) [inline modules/nf-core/zip/tests/main.nf.test:1]:
```suggestion
```

File: modules/nf-core/zip/tests/main.nf.test.snap [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "versions": {
+        "content": [
+            [
+                "versions.yml:md5,721b9a1e22ba1d75fe6a30bb1684cf02"
+            ]
+        ],
```
Comment (sateeshperi) [inline modules/nf-core/zip/tests/main.nf.test.snap:7]: you are not capturing anything else other than versions in the snapshot ?

Comment (JoseEspinosa) [reply modules/nf-core/zip/tests/main.nf.test.snap:7]: I am checking [here](https://github.com/nf-core/modules/blob/8debddd2876a7e079f552fbbae7d01b97911fdc1/modules/nf-core/zip/tests/main.nf.test#L27) that the zip file is produced as the file md5 hash changes. I have seen this in other modules, e.g. https://github.com/nf-core/modules/blob/master/modules/nf-core/salmon/quant/tests/main.nf.test.snap do I have to do something else @sateeshperi ? Thanks!

Comment (sateeshperi) [reply modules/nf-core/zip/tests/main.nf.test.snap:7]: if there is nothing to be done...its fine

**Review by sateeshperi — APPROVED** [review]

File: modules/nf-core/zip/meta.yml [hunk 3]
```diff
@@ -38,6 +38,8 @@ output:
 authors:
   - "@jfy133"
   - "@pinin4fjords"
+  - "@JoseEspinosa"
```
Comment (Emiller88) [inline modules/nf-core/zip/meta.yml:41]:
```suggestion
```

This field is just a legacy #4173 thing. I don't make the rules 🤷🏻

**Review by Emiller88 — COMMENTED** [review]
Just a note.

**Review by SPPearce — APPROVED** [review]

**Pull request merged**